packages/components/src/NavBar/NavMenu.tsx

```diff
@@ -44,6 +48,23 @@ const NavMenuLinksComponent: React.FunctionComponent<NavMenuCloseDrawerProp> = p
           <NavLinkContractAddressesText />
         </NavLink>
       </NavDropDown>
+      <FeatureFlag feature={"boosts-mvp"}>
```

Comment:
not important right now, but next time we use a feature flag we should create a constant for it